apps/supernova/src/lib/createFiltersSlice.test.js

```diff
@@ -288,18 +288,23 @@ describe("createFiltersSlice", () => {
         filterLabels: ["region"],
       }
 
+      const expectedFilters = {
+        region: ["europe"],
+      } // Only region is a valid filter label, we except everything else to be filtered out
+
       const wrapper = ({ children }) => <StoreProvider options={props}>{children}</StoreProvider>
 
-      renderHook(
+      const store = renderHook(
         () => ({
           activeFilters: useActiveFilters(),
         }),
         { wrapper }
       )
 
       expect(spy).toHaveBeenCalledWith(
-        "[supernova]::parseInitialFilters: Some keys of the initialFilters object are not in the labels"
+        "[supernova]::parseInitialFilters: Some keys of the initialFilters object are not valid filter labels. They must be configured as filterLabels first. Using only valid keys."
```

Comment:
```suggestion
        "[supernova]::parseInitialFilters: Some keys of the initialFilters object are not valid filter labels. They must be configured as filterLabels first. Using only valid keys."
        expect.stringContaining("[supernova]::parseInitialFilters: Some keys of the initialFilters object are not valid filter labels.")
```

Reply:
I used to compare with `stringContaining` to avoid to fix the test if the text changes since there is jus one warning for the parseInitialFilters.

Reply:
example:
https://github.com/cloudoperators/juno/blob/2a037feaefa10e480fe599b5053037dd0ec7b453/apps/greenhouse/src/lib/plugin.test.js#L22
while the text is bit longer:
https://github.com/cloudoperators/juno/blob/2a037feaefa10e480fe599b5053037dd0ec7b453/apps/greenhouse/src/lib/plugin.js#L33